synapse/groups/groups_server.py

```diff
@@ -342,6 +342,22 @@ def get_group_profile(self, group_id, requester_user_id):
             raise SynapseError(404, "Unknown group")
 
     @defer.inlineCallbacks
+    def update_group_profile(self, group_id, requester_user_id, content):
+        """Update the group profile
+        """
+        yield self.check_group_is_ours(
+            group_id, and_exists=True, and_is_admin=requester_user_id,
+        )
+
+        profile = {}
+        for keyname in ("name", "avatar_url", "short_description",
+                        "long_description"):
+            if keyname in content:
+                profile[keyname] = content[keyname]
```

Comment:
Maybe check that the values are "strings" otherwise postgresql might become sad when we try to insert them, and sqlite will probably do weird things.